Short description: Adjust CS_PATH return value.
Author(s): Fedora glibc team <glibc@lists.fedoraproject.org>
Origin: PATCH
Upstream status: not-needed

In Fedora we should return only /usr/bin because /bin is just a symlink
to /usr/bin after MoveToUsr transition (which glibc has not really
completed).

IMPORTANT: this patch file is saved with 'latin1' encoding and has a
weird character (ù) in it, for testing purposes.

diff -pruN a/sysdeps/unix/confstr.h b/sysdeps/unix/confstr.h
--- a/sysdeps/unix/confstr.h	2012-12-25 08:32:13.000000000 +0530
+++ b/sysdeps/unix/confstr.h	2014-09-05 20:02:55.698275219 +0530
@@ -1 +1 @@
-#define	CS_PATH	"/bin:/usr/bin"
+#define	CS_PATH	"/usr/bin"
